manifests/install/charts/as-a-second-scheduler/templates/configmap.yaml

```diff
@@ -9,31 +9,39 @@ data:
     kind: KubeSchedulerConfiguration
     leaderElection:
       leaderElect: false
+    clientConnection:
+        kubeconfig: "/etc/kubernetes/scheduler.conf"
     profiles:
+    # user can add as many profiles here. Adding two profiles as an example
     - schedulerName: scheduler-plugins-scheduler
       plugins:
+      # Ensure only the first queueSort plugin is rendered.
+
+  {{- if gt (len $.Values.queueSort) 0 }}
```

Comment:
Wait a second... Does that mean the user has to provide a full values.yaml to override the defaults?

It's a good UX, right? I'd pursue that the user just provides a plugin list, and optionally extra plugin configs. WDYT?

Reply:
That means that if the user specifies a value in the array(queueSort and rest of extensions apart), it will be applied, otherwise, the default will be used. So the user doesn't have to provide all values in the `values.yaml`

Reply:
If a user wants to enable `CapacityScheduling` **only**, what's the Helm command to achieve that?

Reply:
Yes, by running the following:

`helm install scheduler-plugins ./as-a-second-scheduler/ --set extensions.preFilter={"CapacityScheduling"} --set extensions.postFilter={"CapacityScheduling"} --set extensions.reserve={"CapacityScheduling"} --set queueSort=null --set extensions.filter=null --set extensions.score=null --set extensions.permit=null --set extensions.postBind=null`

Reply:
I'm afraid this is a bit verbose... Can we just provide a "plugins" option to make it oriented to users? End-user is better not aware of the extensions settings:

- plugins.enabled="CapacityScheduling" - this should be all the user needs to provide
- (optionally) plugins.disabled="<any default plugins>" - user can also disable plugins enabled by default in the vanilla scheduler

Reply:
> I'm afraid this is a bit verbose... Can we just provide a "plugins" option to make it oriented to users? End-user is better not aware of the extensions settings:
> 
> * plugins.enabled="CapacityScheduling" - this should be all the user needs to provide
> * (optionally) plugins.disabled="" - user can also disable plugins enabled by default in the vanilla scheduler

Alright, now the user can set only one variable `plugins.enabled` :

`helm install scheduler-plugins ./as-a-second-scheduler/ --set "plugins.enabled={"Coscheduling","CapacityScheduling","NodeResourceTopologyMatch","LoadVariationRiskBalancing","TargetLoadPacking","NodeResourcesAllocatable"}"`

Reply:
Kudos! Thanks for the efforts!